pages/src/source/workers/undertaker.md

```diff
@@ -14,7 +14,13 @@ After a short time, graves will decay and eventually disappear. There are [resea
 
 The grave (decayed or not) will hold all the items the citizen had in their inventory at the time of death. The player can right-click the grave to open its inventory and retrieve the items. The Undertaker will walk (run if you've completed the relevant research) toward the grave and retrieve its inventory, then go back to the Graveyard.
 
-Once the Undertaker gets to the Graveyard, they will attempt to revive the deceased citizen. The chance for them to succeed can be increased by researches, the Undertaker's Mana skill, and the level of the Graveyard. The chance is capped at about 10%.
+Once the Undertaker gets to the Graveyard, they will attempt to revive the deceased citizen. The chance for them to succeed can be increased by:
+- researches (1x 1%,1x 2%)
```

Comment:
Rather than having the numbers here, can you link to the research page?

Reply:
I can link it, but for consistence with the other points, I would keep the percentages her to. You dont always want to go to other pages to look up everything. Or do you really think, it should be removed?

Reply:
I still think, it is not so cool, that you have to go to the research page to search for resurrection chances. Especially since they are not sorted by research tree but by column, making it hard to find all the researches for the revive chance. And all the other points have the percentages behind them, so for consistency, I would like to keep the percentages there to.

Reply:
In that case, the formatting would need to change a bit.

`researches (+1% and +2%)`

Reply:
It would be possible (to an extent), links can contain something called a "fragment", you can do this by putting a # after your URL.
What this will achieve is that when landing on the page you will automatically move your viewport down to find the appropriate fragment on the page (element annotated with the matching `id`)

In Markdown headers automatically get ids attached to them (for the research page this would be `second-column`), however this one is reused a couple of times.

We have to improve the research pages so all research item rows will get an id in the elements so we can properly navigate to each one

Reply:
Just so you can see it in action yourself, try navigate to: https://wiki.minecolonies.ldtteam.com/source/systems/research#second-column

Reply:
I think this is fine for now. If we want to link to the various research pages, we can do that later.